src/aics-image-viewer/components/App/index.tsx

```diff
@@ -235,7 +268,6 @@ const App: React.FC<AppProps> = (props) => {
       view3d.updateLuts(image);
       view3d.onVolumeData(image, [channelIndex]);
 
-      view3d.setVolumeChannelEnabled(image, channelIndex, thisChannelSettings.volumeEnabled);
```

Comment:
Is this getting called somewhere else?

Reply:
good catch, not sure why this is removed here.  @frasercl does this look right?